package.json

```diff
@@ -27,8 +27,9 @@
     "react-center-component": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "16.0.0",
+    "react-dom": "16.0.0",
```

Comment:
This will still work with react `15.6`, you should expand the version range to be more permissive.
Something like `15.x || ^16.0.0` would be better.

The react & prop-types peerDependencies should also be more lenient.